### PR TITLE
DUOS 673 Remove user details from darInfo & describeDar refactor

### PR DESCRIPTION
**Pull request by JVThomas**
Addresses [DUOS-673](https://broadinstitute.atlassian.net/browse/DUOS-673)

Basic premise is to extract user details out of darInfo and rely on the data returned from ResearcherProfile. Additional changes also include removing dependency on the modal summary endpoint and instead using the v2 DAR GET and User GET endpoints as well as adding additional helper functions and details to the dataUseTranslation file to define purpose statements and researchTypes so that the back-end does not have to (nor should it, data presentation is a front-role responsibility) 

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] PR is labeled with a Jira ticket number and includes a link to the ticket
- [x] PR is labeled with a security risk modifier [no, low, medium, high] 
- [x] PR describes scope of changes

In all cases:

- [x] Get a minimum of one thumbs worth of review, preferably 2 if enough team members are available
- [x] Get PO sign-off for all non-trivial UI or workflow changes
- [x] Verify all tests go green
- [x] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/libs/ajax.js [hunk 1]
```diff
@@ -251,39 +251,40 @@ export const DAR = {
...
+    darInfo.darCode = rawDar.darCode;
+    darInfo.projectTitle = rawDar.projectTitle;
+    darInfo.institution = rawDar.institutionName;
+    darInfo.department = rawDar.department;
+    darInfo.city = rawDar.city;
+    darInfo.country = rawDar.country;
+    darInfo.status = rawDar.status;
+
+    //NOTE:does rationale need to come from researcher or current user (guessing researcher)?
+    darInfo.hasAdminComment = researcher.rationale != null;
```
Comment (lindojong) [inline src/libs/ajax.js:266]: I'm inclined to believe that rationale should come from the current user? Rationale appears to be the comment made for the reasoning behind a vote approving/denying a request (see all the finalRationale fields across elections and votes). This particular field only seems to be accessed from admin level roles though.

File: src/libs/ajax.js [hunk 2]
```diff
@@ -251,39 +251,40 @@ export const DAR = {
...
-    const profileName = ld.get(ld.head(ld.filter(darInfo.researcherProperties, { 'propertyKey': 'profileName' })), 'propertyValue', "");
-    const piName = ld.get(ld.head(ld.filter(darInfo.researcherProperties, { 'propertyKey': 'piName' })), 'propertyValue', "");
-    darInfo.pi = isThePI ? profileName : piName;
-    darInfo.havePI = havePI || isThePI;
-    darInfo.profileName = profileName;
+
+    darInfo.datasets = rawDar.datasets;
+    darInfo.pi = rawDar.investigator;
+    darInfo.havePI = rawDar.havePI || rawDar.isThePI;
+    //NOTE:Is this meant to display the researcher name?
```
Comment (lindojong) [inline src/libs/ajax.js]: Yes this is the researcher's name

File: src/libs/ajax.js (hunk 1 above)
Comment (rushtong) [reply src/libs/ajax.js:266]: This is correct, rationale is on the user. It has a bit of a complicated history, but in essence, it is what the admin enters for reasoning when approving or denying a user as a researcher.

File: src/libs/dataUseTranslation.js [hunk 3]
```diff
@@ -10,6 +104,86 @@ export const DataUseTranslation = {
    * @param darInfo
    * @returns {{primary: [{code: '', description: ''}], secondary: [{code: '', description: ''}]}}
    */
+
+  //NOTE: backend categorization of purposeStatement/researchType differs from front-end primary/secondary designations
+  //Should the two categories be unified or are they distinct for a reason?
```
Comment (rushtong) [inline src/libs/dataUseTranslation.js]: The back end categorization of `purposeStatement/researchType` is out of date which is why we want to move them to the front end and make them consistent with how we're describing primary/secondary designations.

File: src/libs/dataUseTranslation.js [hunk 4]
```diff
@@ -1,5 +1,99 @@
 import * as fp from 'lodash/fp';
 
+const translations = {
```
Comment (rushtong) [inline src/libs/dataUseTranslation.js:3]: FYI, love this refactor.

File: src/libs/dataUseTranslation.js [hunk 5]
```diff
@@ -1,5 +1,99 @@
...
+    manualReview: false
+  },
+  poa: {
+    code: 'POA',
+    desciption: 'The dataset will be used for the study of Population Origins/Migration patterns.',
+    manualReview: true
+  },
+  diseases: (diseases) => {
+    const diseaseArray = diseases.sort();
+    const diseaseString = diseaseArray.length > 1 ? fp.join('; ')(diseaseArray) : diseaseArray[0];
```
Comment (rushtong) [inline src/libs/dataUseTranslation.js:16]: I don't think you need to check for length here, pretty sure it does the right thing with arrays of any length, even 0.

File: src/libs/ajax.js (hunk 1 above)
Comment (rushtong) [reply src/libs/ajax.js:266]: Just realized my comment was confusing. The rationale here is attached to the researcher that made the data access request, not the currently logged in user.

**Review by rushtong — APPROVED** [review]
Looks good. See one minor comment inline, otherwise 👍

**Comment by JVThomas** [comment]
I recently added commits to correct some errors I made when processing the researcherType statements (The statements under "Types of Research" in ApplicationSummary), so I'd appreciate a re-review of those changes. Thanks!

**Review by rushtong — APPROVED** [review]
LGTM 👍

**Pull request merged**